README.md

```diff
@@ -17,6 +17,11 @@ This is quite slow and can take up to one or two hours but can provide good resu
 
 ### Usage
 
+#### Install required build tools:
+
+-   This project is based upon Node.js. Therefore, there has to be a recent version of Node.js installed on your machine.
```

Comment:
Lets specify a lower limit for the node version.

Reply:
We should Support at least the latest LTS (v18).

Reply:
Ok. Should I explicitely verify that the program runs with version 18?

Reply:
No, mentioning it in the readme as a requirement should be fine.